MAUI/Busy-Indicator/Sizing.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: Sizing in .NET MAUI Busy Indicator control | Syncfusion
+description: Learn here all about Sizing support in Syncfusion .NET MAUI Busy Indicator (SfBusyIndicator) control and more.
+platform: maui
+control: BusyIndicator
+documentation: ug
+---
+# Sizing in .NET MAUI Busy Indicator (SfBusyIndicator)
+
+## Sizing in SfBusyIndicator
+
+Drawing size can be customized in SfBusyIndicator.  `SizeFactor` property can be used to set height and width of the Indicator.  Its default value is 0.5 and  range between 0 to 1.
```

Comment:
Drawing size can be customized in SfBusyIndicator. `SizeFactor` property can be used to set the height and width of the Indicator. Its default value is 0.5 and ranges from 0 to 1.